README.md

```diff
@@ -1 +1,94 @@
-# oms-api
\ No newline at end of file
+# oms-api
+
+# Prerequisite
+Ionic CLI - If you don't have the ionic CLI installed refer [official documentation](https://ionicframework.com/docs/intro/cli) for the installation instructions.
+
+
+# Build Notes (Users)
+
+1. Download the app from [release](https://github.com/hotwax/oms-api/releases) page and extract it.
+2. Go to the app directory.
+3. Run following command to download dependencies  
+    `npm i
+4. Create a `.env` file by taking reference from the `.env.example`.
+5. To run the app in browser use the command: `ionic serve`
```

Comment:
```suggestion
4. Build the package using `npm run build`
```

Reply:
Do the same changes for the build notes (contributors) as well.

Reply:
I have made the suggested change.